site/content/zh-CN/docs/tasks/manage/setup_wait_for_pods_ready.md

```diff
@@ -509,21 +497,20 @@ job2-8-xdqtp    0/1     Pending     0          2s
 job2-9-c4rcl    0/1     Pending     0          2s
 ```
 
-Once `job1` completes, it frees the resources required by `job2` to run its pods
-to make progress. Finally, all jobs complete.
+一旦 `job1` 完成，就会释放 `job2` 运行其 Pod 所需的资源，最终所有作业都能完成。
 
-#### Cleanup
+#### 清理
 
-Clean up the jobs by:
+通过以下命令清理 jobs：
```

Comment:
```suggestion
通过以下命令清理作业：
```

Reply:
should we wait for this comment before approving @my-git9 ?